.travis.yml

```diff
@@ -1,6 +1,26 @@
+sudo: false
+
 language: ruby
+
 rvm:
-- 2.0.0
-- jruby-19mode
-- rbx-2
-script: bundle exec rspec
+  - ruby-1.9.3
+  - ruby-2.0.0
+  - ruby-2.1.7
+  - ruby-2.2.3
```

Comment:
didn't see your comment in time - but I think if it can run with RBX and jRuby -> why not test it.